tools/swift-inspect/Package.swift

```diff
@@ -14,17 +14,23 @@ let package = Package(
         .target(
             name: "swift-inspect",
             dependencies: [
-                "SymbolicationShims",
+                "libswiftInspect",
                 .product(name: "ArgumentParser", package: "swift-argument-parser"),
             ],
             swiftSettings: [
                 .unsafeFlags([
                     "-parse-as-library",
                 ]),
             ]),
+        .target(
+            name: "libswiftInspect",
+            dependencies: [
+                "SymbolicationShims",
+                .product(name: "ArgumentParser", package: "swift-argument-parser"),
+            ]),
```

Comment:
I don't understand this change.  Why are we making this into a library?  This is definitely going to cause some trouble on Windows where we do not yet support static libraries (with SPM).

Reply:
It's so the test can access the swift-inspect code. Is there another way to do that?

Reply:
Ah, I see.  In that case, I think that we should undo the changed for the `internal` to `public` as `@_testable import SiwftInspect` should take care of that.  However, I don't think that it makes sense to make the tests also depend on swift-argument-parser.  We should be able to keep the argument processing in the binary and only expose the internals.